### PR TITLE
incomplete bgp route map for RM_SET_SRC/RM_SET_SRC6

### PR DESCRIPTION
**Pull request by tylerlinp**
**- What I did**
Fix the issue [#3735](https://github.com/Azure/sonic-buildimage/issues/3735)
1. fix the "Loopback0": {}, by modifying minigraph.py(Prince has made the fix in sonic-utilities for db migrate (https://github.com/Azure/sonic-utilities/pull/734))
2. Generate the ip address configuration for setting src to solve configuration recovering failed.

**- How I did it**

**- How to verify it**
test

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by tylerlinp** [comment]
retest vs please

**Comment by akokhan** [comment]
retest vs please

File: dockers/docker-fpm-frr/zebra.interfaces.conf.j2 [hunk 1]
```diff
@@ -39,13 +39,23 @@ link-detect
 {%     endif %}
 {%   endfor %}
 {% endif %}
+{% if lo_ipv4_addrs|length > 0 or lo_ipv6_addrs|length > 0 %}
+interface Loopback0
+{% if lo_ipv4_addrs|length > 0 %}
+    ip address {{ lo_ipv4_addrs[0] }}
+{% endif %}
+{% if lo_ipv6_addrs|length > 0 %}
+    ipv6 address {{ lo_ipv6_addrs[0] | ip }}/{{ lo_ipv6_addrs[0] | prefixlen }}
```
Comment (lguohan) [inline dockers/docker-fpm-frr/zebra.interfaces.conf.j2]: this does not subscript the config db lo address change.

Comment (tylerlinp) [reply dockers/docker-fpm-frr/zebra.interfaces.conf.j2]: Yes, so does all configs in templates, there are several commands referenced the lo address. To follow changes, a lot of work to do. I think it is a new requirement.

**Comment by tylerlinp** [comment]
retest vs please

**Comment by prsunny** [comment]
hi Tyler, can you please separate the minigraph changes to another PR as we would need that to unblock `load minigraph`.

**Comment by tylerlinp** [comment]
@prsunny OK. PR #3792

**Comment by lguohan** [comment]
obsoleted by https://github.com/Azure/sonic-buildimage/commit/057ced03910a78f02402828a6d7000f513fc2a30

**Pull request closed without merging**